test/e2e/demo-storage-plugin/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/jaegertracing/jaeger-operator/test/e2e/demo-storage-plugin
+
+go 1.14
+
+require (
+	github.com/hashicorp/go-hclog v0.13.0
+	github.com/jaegertracing/jaeger v1.17.1
```

Comment:
This should match what we have in `versions.txt`

Reply:
How? Fail the CI build if it is different?

Reply:
Not needed, just make sure we have a `sed` in the `release.sh` script.